docs_user/modules/proc_migrating-ceph-mds.adoc

```diff
@@ -205,15 +212,15 @@ Standby daemons:
 . To migrate MDS to the right nodes, set the MDS affinity that manages the MDS failover:
 ifeval::["{build}" != "downstream"]
 [NOTE]
-It is possible to elect as "active" a dedicated MDS for a particular file system. To configure this preference, `CephFS` provides a configuration option for MDS called `mds_join_fs` which enforces this affinity.
-When failing over MDS daemons, a cluster’s monitors will prefer standby daemons with `mds_join_fs` equal to the file system name with the failed rank. If no standby exists with `mds_join_fs` equal to the file system name, it will choose an unqualified standby as a replacement.
+It is possible to elect as "active" a dedicated MDS for a particular file system. To configure this preference, `CephFS` provides a configuration option for MDS called `mds_join_fs`, which enforces this affinity.
+When failing over MDS daemons, a cluster’s monitors prefer standby daemons with `mds_join_fs` equal to the file system name with the failed rank. If no standby exists with `mds_join_fs` equal to the file system name, it chooses an unqualified standby as a replacement.
 endif::[]
 +
 ----
-ceph config set mds.mds.cephstorage-0.fqcshx mds_join_fs cephfs
+$ ceph config set mds.mds.cephstorage-0.fqcshx mds_join_fs cephfs
 ----
 
-. Remove the labels from Controller nodes and force the MDS failover to the
+. Remove the labels from the Controller nodes and force the MDS failover to the
 target node:
 +
 ----
```

Comment:
Regarding the sentence that says, "The switch happens behind the scenes, and the new active MDS is the one that you set through the `mds_join_fs` command." 

Should this be conditionalized for upstream? There is a note in this procedure that mentions `mds_join_fs` that is upstream only, and without that context, this sentence could be confusing.

Reply:
I think we can remove the conditional and keep that context downstream as well. We need this command in both cases, so it's good to have context.